packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -440,6 +452,36 @@ export default class Reachability {
     });
   }
 
+  /**
+   * Adds public IPs (client media IPs)
+   * @param {RTCPeerConnection} peerConnection
+   * @param {string} publicIP
+   * @returns {void}
+   */
+  protected addPublicIPs(peerConnection: RTCPeerConnection, publicIP?: string | null) {
+    const {CLOSED} = CONNECTION_STATE;
+
+    if (peerConnection.connectionState === CLOSED) {
+      LoggerProxy.logger.log(
+        `Reachability:index#addPublicIPs --> Attempting to set publicIP of ${publicIP} on closed peerConnection.`
+      );
+    }
+
+    if (publicIP) {
+      // @ts-ignore
```

Comment:
maybe instead of so many ts-ignore we could cast the peerConnection to an object that has `publicIPs` property once at the beginning?